conf.yaml

```diff
@@ -60,6 +60,7 @@ targets:
     href: https://arstechnica.com
     hosts:
       - arstechnica.com
+      - cnd.arstechnica.net
```

Comment:
you were trying to add cdn.arstechnica.net right?

Reply:
Yep. My fault.

Reply:
Fixed in https://github.com/andir/ipv6.watch/commit/7fb54f4c939d014bb318fbb6eac67b17a994b6fb

Thanks!